### PR TITLE
Merge fixes

### PR DESCRIPTION
**Pull request by fieldse**
Hi George,
Discussed fixes here. Please give it a look-over and let me know if anything needs changes before merge.

### CODE REVIEW

File: b2blaze/models/file_list.py [hunk 1]
```diff
@@ -59,53 +90,140 @@ def _update_files_list(self, retrieve=False):
...
+            'bucketId': self.bucket.bucket_id
+        }
+
+        response = self.connector.make_request(path, method='post', params=params)
+        file_json = response.json()
+
+        # Handle errors and empty files
+        if not response.status_code == 200:
+            raise B2Exception.parse(response)
+        if not len(file_json['files']) > 0:
```
Comment (42B) [inline b2blaze/models/file_list.py:209]:
```python
if len(file_json['files']) == 0:

# or

if not len(file_json['files']):
```

Comment (fieldse) [reply b2blaze/models/file_list.py:209]: @42B Nice, thanks for the improvement! Method #2 is nice, very clean. Those if len(.... ) statements always seem a bit unaesthetic to me.

**Comment by sibblegp** [comment]
Tests are currently failing: https://scrutinizer-ci.com/g/sibblegp/b2blaze/inspections/30965dee-3111-4873-a6ce-7c5b480b1c5e/log

**Comment by fieldse** [comment]
Hi George.
Looking at the logs to try to figure out what's going on. Hard to say from here.

Tests are all passing on my end: Python 3.6

```
collected 23 items                                                                                                     
                                                                                                                       
tests.py::TestB2::test_create_b2_instance PASSED                                                                [  4%] 
tests.py::TestB2::test_create_bucket PASSED                                                                     [  8%] 
tests.py::TestB2::test_get_bucket PASSED                                                                        [ 13%]
tests.py::TestB2::test_get_all_buckets PASSED                                                                   [ 17%]
tests.py::TestB2::test_get_nonexistent_bucket PASSED                                                            [ 21%]
tests.py::TestB2::test_create_file_and_retrieve_by_id PASSED                                                    [ 26%]
tests.py::TestB2::test_direct_upload_file PASSED                                                                [ 30%]
tests.py::TestB2::test_get_all_files PASSED                                                                     [ 34%]
tests.py::TestB2::test_get_all_file_versions PASSED                                                             [ 39%]
tests.py::TestB2::test_get_file_by_name PASSED                                                                  [ 43%]
tests.py::TestB2::test_get_file_by_id PASSED                                                                    [ 47%]
tests.py::TestB2::test_get_file_versions PASSED                                                                 [ 52%]
tests.py::TestB2::test_bucket_get_file_versions_by_name PASSED                                                  [ 56%]
tests.py::TestB2::test_bucket_get_file_versions_by_id PASSED                                                    [ 60%]
tests.py::TestB2::test_get_file_doesnt_exist PASSED                                                             [ 65%]
tests.py::TestB2::test_download_file PASSED                                                                     [ 69%]
tests.py::TestB2::test_download_url PASSED                                                                      [ 73%]
tests.py::TestB2::test_hide_file PASSED                                                                         [ 78%]
tests.py::TestB2::test_delete_file_version PASSED                                                               [ 82%]
tests.py::TestB2::test_delete_all_file_versions PASSED                                                          [ 86%]
tests.py::TestB2::test_delete_non_empty_bucket PASSED                                                           [ 91%]
tests.py::TestB2::test_bucket_delete_all_files PASSED                                                           [ 95%]
tests.py::TestB2::test_delete_bucket PASSED                                                                     [100%]

============================================= 23 passed in 78.80 seconds ==============================================
```

**Comment by fieldse** [comment]
Should not make any difference, but my test suite is vanilla pytest.
We need to get more information on the failing tests.
Can you run locally and see what happens?

**Pull request merged**